packages/ursula/src/front.js

```diff
@@ -21,52 +21,71 @@ export default function (part) {
   store.set('xScale', utils.stretchToScale(options.fabricStretch))
 
   // Design pattern here
+  
+  // determine height of front part: use cross seam (and cross seam front) if selected and available
+  if (options.useCrossSeam && measurements.crossSeam) {
+	store.set('crossSeam',measurements.crossSeam)
+  } else { // use original approximation: front and back are roughly waistToUpperLeg high, plus gusset length
+	  store.set('crossSeam',measurements.waistToUpperLeg * (1 + options.backToFrontLength) + options.gussetLength * measurements.seat)
+  }
+  // optionally use crossSeamFront to determine relative length of front and back
+  if (options.useCrossSeam && measurements.crossSeamFront) { // subtract half the gusset length from cross seam front
+    store.set('frontHeight',measurements.crossSeamFront - measurements.seat*0.5*options.gussetLength)
+  } else { // subtract gusset length, divide by roughly 2
+	store.set('frontHeight',(store.get('crossSeam') - options.gussetLength * measurements.seat)/(1 + options.backToFrontLength))
+  }
+	  
```

Comment:
There is xtra blank line here containing a tab character. I suggest simply removing the line
```suggestion
  }
```

Reply:
(also done)